vl-proza-message.src.js

```diff
@@ -37,6 +39,14 @@ export class VlProzaMessage extends VlElement(HTMLElement) {
                 <slot></slot>
             </div>
         `);
+        const id = "vl-proza-message-toaster";
+        if (document.getElementById(id) == null) {
+        	const toaster = document.createElement('div', {is: 'vl-toaster'});
+        	toaster.setAttribute("top-right", "");
+        	toaster.setAttribute("id", id);
+        	document.body.appendChild(toaster);
+        }
+        this._toaster = document.getElementById(id);
```

Comment:
Om de constructor leesbaar(der) te houden, zou ik dit verplaatsen naar één of meerdere functies.